src/app/services/serviceConfig.js

```diff
@@ -70,8 +97,10 @@ const buildCurrentServiceModel = async (req) => {
 
 const getServiceConfig = async (req, res) => {
   try {
-    if (req.session.serviceConfigurationChanges && req.query.action !== ACTIONS.AMEND_CHANGES) {
+    if (req.query.action !== ACTIONS.AMEND_CHANGES) {
       req.session.serviceConfigurationChanges = {};
```

Comment:
log.info amend changes action please

Reply:
the query parameter is used to populate the fields with the values from Review service configuration page when the user is coming back from Review service configuration page - there will be no need to log.info this action